scripts/run-opencode-sentry-fix.sh

```
@@ -0,0 +1,361 @@
+
+FULL_CONTEXT="${SENTRY_CONTEXT}GitHub Issue #${ISSUE_NUMBER}: ${ISSUE_TITLE}
+
+Issue URL: ${ISSUE_URL}
+
+Issue Description:
+${ISSUE_BODY}
+
+=== CRITICAL INSTRUCTIONS ===
+You MUST follow the erxes Agent Manifest Protocol (AMP) defined in this repository.
+
+1. **READ `.agents/manifest.yaml` FIRST** - This is mandatory. Understand the rule layers, skills, and protocol.
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)